### ISSUE TITLE
Compile error for cloud_firestore on ios

### ISSUE THREAD

**Issue by chiuki** [flutter/flutter#18689]
## Steps to Reproduce

1. `flutter create myapp; cd my app`
2. Add `cloud_firestore: ^0.7.3` to `pubspec.yaml`

### flutter run

```
Launching lib/main.dart on iPhone X in debug mode...
Starting Xcode build...                                          
Xcode build done.                                            5.5s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **
    
Xcode's output:
↳
    === BUILD TARGET FirebaseFirestore OF PROJECT Pods WITH CONFIGURATION Debug ===
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:36:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1beta1/Firestore.pbrpc.h:18:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1beta1/Firestore.pbobjc.h:27:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:44:
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBWellKnownTypes.h:40:10: warning: non-portable path to file '<protobuf/Any.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Any.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Any.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBWellKnownTypes.h:41:10: warning: non-portable path to file '<protobuf/Duration.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Duration.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Duration.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBWellKnownTypes.h:42:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:36:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1beta1/Firestore.pbrpc.h:18:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1beta1/Firestore.pbobjc.h:27:
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:55:10: warning: non-portable path to file '<protobuf/Any.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Any.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Any.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:56:10: warning: non-portable path to file '<protobuf/Api.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Api.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Api.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:57:10: warning: non-portable path to file '<protobuf/Duration.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Duration.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Duration.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:58:10: warning: non-portable path to file '<protobuf/Empty.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Empty.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Empty.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:59:10: warning: non-portable path to file '<protobuf/FieldMask.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/FieldMask.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/FieldMask.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:60:10: warning: non-portable path to file '<protobuf/SourceContext.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/SourceContext.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/SourceContext.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:61:10: warning: non-portable path to file '<protobuf/Struct.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Struct.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Struct.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:62:10: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Timestamp.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Timestamp.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:63:10: warning: non-portable path to file '<protobuf/Type.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Type.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Type.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/Headers/Private/Protobuf/GPBProtocolBuffers.h:64:10: warning: non-portable path to file '<protobuf/Wrappers.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
     #import <Protobuf/Wrappers.pbobjc.h>
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
             <protobuf/Wrappers.pbobjc.h>
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:36:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1beta1/Firestore.pbrpc.h:57:13: warning: non-portable path to file '<protobuf/Empty.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
        #import <Protobuf/Empty.pbobjc.h>
                ^~~~~~~~~~~~~~~~~~~~~~~~~
                <protobuf/Empty.pbobjc.h>
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Protos/objc/google/firestore/v1beta1/Firestore.pbrpc.h:62:13: warning: non-portable path to file '<protobuf/Timestamp.pbobjc.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
        #import <Protobuf/Timestamp.pbobjc.h>
                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                <protobuf/Timestamp.pbobjc.h>
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:17:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/credentials_provider.h:24:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/token.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/hard_assert.h:22:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:67:9: error: call to constructor of 'firebase::firestore::util::FormatArg' is ambiguous
          : FormatArg{std::forward<T>(value), internal::FormatChoice<0>{}} {
            ^        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:170:46: note: in instantiation of function template specialization 'firebase::firestore::util::FormatArg::FormatArg<NSString *const __strong &>' requested here
          format, {static_cast<const FormatArg&>(args).Piece()...});
                                                 ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:220:5: note: in instantiation of function template specialization 'firebase::firestore::util::StringFormat<NSString *, NSDictionary<NSString *,NSString *> *>' requested here
        LOG_DEBUG("RPC %s returned headers (whitelisted): %s", rpcName,
        ^
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:45:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/log.h:46:37: note: expanded from macro 'LOG_DEBUG'
          std::string _message = _util::StringFormat(__VA_ARGS__); \
                                        ^
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:17:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/credentials_provider.h:24:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/token.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/hard_assert.h:22:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:99:3: note: candidate constructor
      FormatArg(Class object, internal::FormatChoice<0>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:110:3: note: candidate constructor
      FormatArg(id object, internal::FormatChoice<0>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:137:3: note: candidate constructor [with T = NSString]
      FormatArg(T* pointer_value, internal::FormatChoice<3>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:146:3: note: candidate constructor [with T = NSString *const __strong &]
      FormatArg(T&& value, internal::FormatChoice<4>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:67:9: error: call to constructor of 'firebase::firestore::util::FormatArg' is ambiguous
          : FormatArg{std::forward<T>(value), internal::FormatChoice<0>{}} {
            ^        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:170:46: note: in instantiation of function template specialization 'firebase::firestore::util::FormatArg::FormatArg<NSDictionary<NSString *,NSString *> *const __strong &>' requested here
          format, {static_cast<const FormatArg&>(args).Piece()...});
                                                 ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:220:5: note: in instantiation of function template specialization 'firebase::firestore::util::StringFormat<NSString *, NSDictionary<NSString *,NSString *> *>' requested here
        LOG_DEBUG("RPC %s returned headers (whitelisted): %s", rpcName,
        ^
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:45:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/log.h:46:37: note: expanded from macro 'LOG_DEBUG'
          std::string _message = _util::StringFormat(__VA_ARGS__); \
                                        ^
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:17:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/credentials_provider.h:24:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/token.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/hard_assert.h:22:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:99:3: note: candidate constructor
      FormatArg(Class object, internal::FormatChoice<0>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:110:3: note: candidate constructor
      FormatArg(id object, internal::FormatChoice<0>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:137:3: note: candidate constructor [with T = NSDictionary<NSString *,NSString *>]
      FormatArg(T* pointer_value, internal::FormatChoice<3>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:146:3: note: candidate constructor [with T = NSDictionary<NSString *,NSString *> *const __strong &]
      FormatArg(T&& value, internal::FormatChoice<4>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:67:9: error: call to constructor of 'firebase::firestore::util::FormatArg' is ambiguous
          : FormatArg{std::forward<T>(value), internal::FormatChoice<0>{}} {
            ^        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:170:46: note: in instantiation of function template specialization 'firebase::firestore::util::FormatArg::FormatArg<NSError *const __strong &>' requested here
          format, {static_cast<const FormatArg&>(args).Piece()...});
                                                 ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:242:28: note: in instantiation of function template specialization 'firebase::firestore::util::StringFormat<NSError *>' requested here
                               LOG_DEBUG("RPC CommitRequest completed. Error: %s", error);
                               ^
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:45:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/log.h:46:37: note: expanded from macro 'LOG_DEBUG'
          std::string _message = _util::StringFormat(__VA_ARGS__); \
                                        ^
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:17:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/credentials_provider.h:24:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/auth/token.h:23:
    In file included from /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/hard_assert.h:22:
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:99:3: note: candidate constructor
      FormatArg(Class object, internal::FormatChoice<0>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:110:3: note: candidate constructor
      FormatArg(id object, internal::FormatChoice<0>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:137:3: note: candidate constructor [with T = NSError]
      FormatArg(T* pointer_value, internal::FormatChoice<3>)
      ^
    /Users/chiuki/demo/myapp/ios/Pods/FirebaseFirestore/Firestore/core/src/firebase/firestore/util/string_format.h:146:3: note: candidate constructor [with T = NSError *const __strong &]
      FormatArg(T&& value, internal::FormatChoice<4>)
      ^
    15 warnings and 3 errors generated.
Could not build the application for the simulator.
Error launching application on iPhone X.
```

### flutter analyze
```
Analyzing myapp...                                               
No issues found! (ran in 10.8s)
```

### flutter doctor

```
[✓] Flutter (Channel beta, v0.5.1, on Mac OS X 10.12.6 16G1314, locale en-US)
    • Flutter version 0.5.1 at /Users/chiuki/dev/flutter
    • Framework revision c7ea3ca377 (3 weeks ago), 2018-05-29 21:07:33 +0200
    • Engine revision 1ed25ca7b7
    • Dart version 2.0.0-dev.58.0.flutter-f981f09760

[✓] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /Users/chiuki/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • ANDROID_HOME = /Users/chiuki/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)
    • All Android licenses accepted.

[✓] iOS toolchain - develop for iOS devices (Xcode 9.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.2, Build version 9C40b
    • ios-deploy 1.9.2
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 24.1.1
    • Dart plugin version 173.4700
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1024-b01)

[✓] Connected devices (1 available)
    • iPhone X • 176E81F7-A5B3-42C3-A453-33378DB82D86 • ios • iOS 11.2 (simulator)

• No issues found!
```

**Comment by g-viet** [comment]
Same here !

**Comment by zoechi** [comment]
Can you try `pod install` in the `ios` directory?

**Comment by g-viet** [comment]
It works fine ! For me.

**Comment by chiuki** [comment]
@zoechi I tried `pod install` in the `ios` directory and got this:

[!] Automatically assigning platform `ios` with version `8.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Tried uncommenting `platform :ios, '9.0'` in the Podfile and `flutter run` still gives me the same error.

**Comment by matteobart** [comment]
Having the same problem here!

**Comment by adriancmurray** [comment]
I'm running into this exact same problem. I'm betting a lot of people have this problem but don't run into it until they do a flutter clean. Previous builds keep working. It's not until you rebuild that you see a problem. That seemed to be the case with #12725 as well.

**Comment by zoechi** [comment]
@adriancmurray have you tried `pod install` or `pod update`?

**Comment by adriancmurray** [comment]
@zoechi I've done everything. Flutter doctor, upgrade, pod install, I even cloned the project without any of the generated content to see if that helped. Same problem.

**Comment by adriancmurray** [comment]
@zoechi errors I'm getting: 

Failed to build iOS app
Error output from Xcode build:
↳
Could not build the application for the simulator.
Error launching application on iPhone 8 Plus.
    ** BUILD FAILED **

.../ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:243:30: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
                                 _credentials->InvalidateToken();
^
                                 self->

.../ios/Pods/FirebaseFirestore/Firestore/Source/Remote/FSTDatastore.mm:280:38: warning: block implicitly retains 'self'; explicitly mention 'self' to indicate this is intended behavior [-Wimplicit-retain-self]
                                         _credentials->InvalidateToken();
                                         ^
                                         self->

2 warnings generated.
    === BUILD TARGET Runner OF PROJECT Runner WITH CONFIGURATION Debug ===
    ld: framework not found App
    clang: error: linker command failed with exit code 1 (use -v to see invocation)

**Comment by zoechi** [comment]
That error looks quite differently then the one this issue is about. 
Perhaps better to create a new issue for that.

**Comment by adriancmurray** [comment]
@zoechi I know the issues read differently but I'm thinking this has to do with Swift and Firebase not playing nice together. This has propagated other issues similar to this. There seems to be a conflict with cocoa pods and firebase when Swift is allowed. I've tried everything mentioned here #17469
as well.

**Comment by adriancmurray** [comment]
@zoechi I just did another rebuild and deselected Swift during the setup process. It works now. I think there is a issue with Swift and firebase plugins for Flutter. @chiuki you may want to remove Swift as an option. That is, if you can get away with that.

**Comment by chiuki** [comment]
@adriancmurray Sure, I don't need Swift, since I'm doing everything in dart right now. How do I remove it?

**Comment by adriancmurray** [comment]
@chiuki there might be an easier method but you have to create a new project and when you get to this screen:

<img width="339" alt="screen shot 2018-06-28 at 12 07 12 pm" src="https://user-images.githubusercontent.com/39349880/42052379-da4e87c4-7acb-11e8-9943-43f13205573b.png">

Just make sure that you have Swift deselected. This is using Android Studio. Then you move over your existing code, settings and assets to the new project. Which means setting up a new iOS app in firebase. Let me know if that works out for you.

**Comment by chiuki** [comment]
I fixed my build! This is what I did:

* `brew install ruby`
* `pod update` in the `ios` directory

This updates `FirebaseFirestore` from `0.12.3` to `0.12.4`, along with other firebase dependencies.

The `ruby` update is needed because the default version of ruby in Sierra runs an old version of `openssl`, which leads to this `pod update` error:

    [!] Failed to connect to GitHub to update the CocoaPods/Specs specs repo - Please check if you are offline, or that GitHub is down

More info [here](https://stackoverflow.com/questions/49166609/updating-pod-is-not-working).

I also updated `openssl` but I don't think that was needed. No need to remove Swift.
